audio/music/cinnabarmansion.asm

```diff
@@ -94,11 +89,10 @@ Music_CinnabarMansion_branch_7ed6c::
 	note A#, 4
 	sound_ret
 
-
 Music_CinnabarMansion_Ch3::
 	note_type 12, 1, 1
-
-Music_CinnabarMansion_branch_7ed80::
+.mainloop:
+.introloop:
```

Comment:
Why the `.mainloop` and `.introloop` here? When in `Music_CinnabarMansion_Ch1::` you only used `.introloop`.

Reply:
Nice catch! `Music_CinnabarMansion_Ch1` should have a `.mainloop` too.